hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoGetAccountRecordsHandlerTest.java

```diff
@@ -0,0 +1,314 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.token.impl.test.handlers;
+
+import static com.hedera.hapi.node.base.ResponseCodeEnum.INVALID_TRANSACTION;
+import static com.hedera.hapi.node.base.ResponseCodeEnum.OK;
+import static com.hedera.hapi.node.base.ResponseType.ANSWER_ONLY;
+import static com.hedera.hapi.node.base.ResponseType.ANSWER_STATE_PROOF;
+import static com.hedera.hapi.node.base.ResponseType.COST_ANSWER;
+import static com.hedera.node.app.spi.fixtures.workflows.ExceptionConditions.responseCode;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mock.Strictness.LENIENT;
+import static org.mockito.Mockito.mock;
+
+import com.hedera.hapi.node.base.AccountID;
+import com.hedera.hapi.node.base.QueryHeader;
+import com.hedera.hapi.node.base.ResponseCodeEnum;
+import com.hedera.hapi.node.base.ResponseHeader;
+import com.hedera.hapi.node.base.ResponseType;
+import com.hedera.hapi.node.base.Transaction;
+import com.hedera.hapi.node.token.CryptoGetAccountRecordsQuery;
+import com.hedera.hapi.node.token.CryptoGetAccountRecordsResponse;
+import com.hedera.hapi.node.token.CryptoGetStakersQuery;
+import com.hedera.hapi.node.transaction.Query;
+import com.hedera.hapi.node.transaction.TransactionRecord;
+import com.hedera.node.app.service.token.ReadableAccountStore;
+import com.hedera.node.app.service.token.impl.handlers.BaseCryptoHandler;
+import com.hedera.node.app.service.token.impl.handlers.CryptoGetAccountRecordsHandler;
+import com.hedera.node.app.service.token.impl.test.handlers.util.CryptoHandlerTestBase;
+import com.hedera.node.app.spi.records.RecordCache;
+import com.hedera.node.app.spi.workflows.PreCheckException;
+import com.hedera.node.app.spi.workflows.QueryContext;
+import com.hedera.pbj.runtime.io.buffer.Bytes;
+import java.util.List;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class CryptoGetAccountRecordsHandlerTest extends CryptoHandlerTestBase {
+    @Mock(strictness = LENIENT)
+    private QueryContext context;
+
+    @Mock(strictness = LENIENT)
+    private RecordCache recordCache;
+
+    private CryptoGetAccountRecordsHandler subject;
+
+    @BeforeEach
+    public void setUp() {
+        super.setUp();
+        subject = new CryptoGetAccountRecordsHandler(recordCache);
+    }
+
+    @SuppressWarnings("DataFlowIssue")
+    @Test
+    void extractHeader_nullArg() {
```

Comment:
nit:
I don't think I've seen this naming pattern anywhere else in the codebase.
Might be worth matching the style in other unit tests for token service.

Reply:
Agreed, I haven't seen it. However, I haven't been able to discern what the naming pattern is either 😂 I'll go take a look and adjust

Reply:
Done